xamarin/Xamarin.Forms
# [Android] Fix issue updating background brush

### Description of Change ###

Fix issue updating background brush from `SolidColorBrush` to `GradientBrush`.

### Issues Resolved ### 

- fixes #13017 

### API Changes ###

 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

![fix13017](https://user-images.githubusercontent.com/6755973/100914205-06195780-34d3-11eb-9227-b893830c56ab.gif)


### Testing Procedure ###
Launch Core Gallery and navigate to the issue 13017. Switch between both Backgrounds several times. If the change works as expected, the test has passed.

### PR Checklist ###
